src/main/kotlin/org/jetbrains/research/ml/codetracker/server/TrackerQueryExecutor.kt

```diff
@@ -124,6 +127,7 @@ object TrackerQueryExecutor : QueryExecutor() {
                 // Throw an error to show the error UI Pane
                 throw IllegalStateException("Unsuccessful server response")
             } else {
+                // Todo: should we delete tmp file with the filtered data?
```

Comment:
If it's not replaced with the new one each time we send data, we definitely should. But I thought it's replaced?